### PR TITLE
Updated wmts and added the automated tests

### PR DESCRIPTION
**Pull request by TonHai1111**
Updated wmts.py to work with the new syntax of new Titiler. Updated the instructions in the Readme. Added some automated tests.

### CODE REVIEW

File: api/endpoints/wmts.py [hunk 1]
```diff
@@ -63,14 +63,23 @@ def gen_mosaic_url(params={}):
     ext = params.get('ext') if params.get('ext') else 'png'
     color_map = params.get('color_map') if params.get('color_map') else 'schwarzwald'
     rescale = params.get('rescale') if params.get('rescale') else '-1,1'
-    return settings.TILER_ENDPOINT + '/mosaic/' + \
+    bidx = params.get('bidx') if params.get('bidx') else '1'
+    #return settings.TILER_ENDPOINT + '/mosaic/' + \
```
Comment (abarciauskas-bgse) [inline api/endpoints/wmts.py:67]: remove this commented code?

File: api/endpoints/wmts.py [hunk 2]
```diff
@@ -161,12 +174,15 @@ def get(self, z, x, y, ext, color_map=None, rescale=None):
 
 
 def get_mosaic_tilejson(urls_query_string=''):
-    mosaic_tilejson_url = settings.TILER_ENDPOINT + '/mosaic/tilejson.json?urls=' + urls_query_string
+    #mosaic_tilejson_url = settings.TILER_ENDPOINT + '/mosaic/tilejson.json?urls=' + urls_query_string
```
Comment (abarciauskas-bgse) [inline api/endpoints/wmts.py:177]: Remove commented code in this file, thanks!

Comment (abarciauskas-bgse) [reply api/endpoints/wmts.py:177]: And elsewhere in this PR!

**Review by abarciauskas-bgse — CHANGES_REQUESTED** [review]
Thanks @TonHai1111 , looks good so far thanks for adding tests!!! 👏🏽  I will try and test it out myself but if you were able to deploy the `maap-api-nasa` locally and get it working that's great!

File: api/endpoints/wmts.py (hunk 1 above)
Comment (TonHai1111) [reply api/endpoints/wmts.py]: Removed!

File: api/endpoints/wmts.py (hunk 2 above)
Comment (TonHai1111) [reply api/endpoints/wmts.py]: Fixed! Thanks!

**Comment by TonHai1111** [comment]
> Thanks @TonHai1111 , looks good so far thanks for adding tests!!! 👏🏽 I will try and test it out myself but if you were able to deploy the `maap-api-nasa` locally and get it working that's great!

**Review by abarciauskas-bgse — APPROVED** [review]
This looks great @TonHai1111 ! I got it to work on my local machine as well. @bsatoriu do you have time to review it?

I also opened a PR for some more deployment instruction refinements but that's non-blocking for this PR.

**Comment by TonHai1111** [comment]
Cool! Thank you Aimee! I wait a bit for Brian before doing the merge. I will take a look to that PR as well!

**Pull request merged**